javascripts/main.js

```diff
@@ -0,0 +1,23 @@
+$(document).ready(function() {
+  function searchMovie(movieName){
+    $.ajax({
+      type: 'GET',
+      url: `https://api.themoviedb.org/3/search/company?api_key=380b0e09e540c1e5680debdf9ad794f3&page=1&query=${movieName}`,
+      success: function(data) {
+
+         movies.innerHTML=""
+         $.each(data.results, function(i, data){
+            console.log('movies::', movies);
+            movies.append( data.name )
+
+         })
+      }
+    })
+  }
+  document.getElementById('movie-submit-button')
```

Comment:
For consistency, if you wanted to continue using jquery (since you've already loaded it and are using the jquery functions above) you might have wanted to use 
```
$('#movie-submit-button').get(0)
  .addEventListener('click', function(event){
// ...
```

or perhaps the jquery function:
```
('#movie-submit-button')
  .click( function(event){ 
// ...
```